pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -104,6 +104,10 @@ const (
 	AnnotationFalse    = "false"
 )
 
+const (
+	errValueCastFailed = "value of annotation %q can't be casted: %s"
+)
```

Comment:
I'm not that keen on the constant format string here, because the reader has to take an extra step to read the format string each time. I'd probably either just use the format literal everywhere, or wrap it in a helper like `func parseError(annotation string, err error) error)`.